### ISSUE TITLE
Baml exception error

### ISSUE THREAD

**Issue by bprg** [mkaring/ConfuserEx#1]
[WpfApp1.zip](https://github.com/mkaring/ConfuserEx/files/2278811/WpfApp1.zip)

```
<Window x:Class="WpfApp1.MainWindow"
        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
        xmlns:local="clr-namespace:WpfApp1"
        mc:Ignorable="d"
        Title="MainWindow" Height="450" Width="800">
    <Grid>
        <TextBlock>[123456</TextBlock>
    </Grid>
</Window>
```

**Comment by mkaring** [comment]
Thank you. I am able to reproduce the error. I'll fix that asap.

**Comment by mkaring** [comment]
@bprg Could you please confirm that [this build](https://ci.appveyor.com/project/mkaring/confuserex/build/30/artifacts "ConfuserEx 30 - AppVeyor") fixes your issue?

**Comment by bprg** [comment]
This issue is fixed, Thank you.

**Comment by bprg** [comment]
Another issue with Comma and Close Bracket.
`<TextBlock>,]</TextBlock>`

I think the PropertyPath Parser should be rewritten.
https://referencesource.microsoft.com/#PresentationFramework/src/Framework/MS/Internal/Data/PathParser.cs

**Comment by mkaring** [comment]
I agree. The thing is that I am not sure why it is even executed for those text blocks. I have yet to find reason why there would be a reference to a property in a plain text entry in the XAML.

**Comment by mkaring** [comment]
I found the reason... there are specific cases when a path to the property is stored in the inner text, that occur for some control if they are directly data bound to XML data.

I am ripping of the implementation of the parser by Microsoft now to get parsing this stuff working properly.

**Comment by mkaring** [comment]
I hope it works significantly better now. I basically copied the original parser from the .NET Framework. So this one should be able to handle the paths properly without problems. The new version will be available in [this build](https://ci.appveyor.com/project/mkaring/confuserex/build/34/artifacts "ConfuserEx 34 - AppVeyor") once it is done.

@bprg Can you verify that the new version works for you?

**Comment by bprg** [comment]
Works properly, Thanks.

**Comment by mkaring** [comment]
Very well. Merged it into the master.
